experimental/torch_xla2/docs/support_a_new_model.md

```diff
@@ -0,0 +1,157 @@
+# Run a model under torch_xla2
+
+Supporting a new model in torch_xla2 means
+having this model run using torch_xla2 and succeeds.
+
+A model usually consists of executing a list of torch ops
+on a set of tensors (i.e. the parameters and inputs) and 
+produce a new tensor(s). These ops should just work.
+
+However, there are cases that the model doesn't run on
+torch_xla2, because:
+
+1. Some op it needs is not implemented.
+2. Some op it needs is implemented incorrectly
+3. There are some non-torch-op code that interacts with torch_xla2 in a non-friendly matter.
+
+Here we present few steps to attempt to fix the related issues.
+
+# Step 1. Attempt to run the model
+
+To run a model under torch_xla2, the first step is to 
+instantiate the model and run it under normal torch.
+This usually means eager mode torch CPU. (NOTE: for large
+ models, it's recommended to make a model of equal architecture but smaller, by setting fewer layers / dim sizes; OR, use GPU
+so that it can run reasonably fast).
+
+In this example, we will use `BERT_pytorch` model from 
+torchbench.
+
+## Install torchbench and instantiate a the model
+
+```bash
+git clone https://github.com/pytorch/benchmark.git torchbench
+cd torchbench
+pip install torchvision torchaudio
+pip install -e .
+```
+Now, torchbench is installed, now we need to download 
+the model.
+
+```
+python install.py BERT_pytorch
+```
+
+NOTE: if you run `python install.py` without positional args 
+it will download ALL the 100+ models which can take sometime.
+
+Now, let's verify that the model is there by importing it in python.
+
+```python
+import torchbenchmark.models.BERT_pytorch
+
+model, sample_inputs = torchbenchmark.models.BERT_pytorch.Model(
+  test='eval', device='cpu'
+)
+
+print(model(*sample_inputs))
+```
+
+If the above succeeds, then the model is ready.
+
+# Attempt to run the model in torchxla2
+
+To run the model in torch_xla2, we need to do 2 things:
+1. Move the model's weight to XLA device (i.e. XLA tensors)
+2. Move the sample_inputs to XLA device (i.e. XLA tensors)
+
+The API for the above is the `to_xla` method on `Environment` class.
+To get the current environment, one can use `torch_xla2.default_env()`.
+
+i.e.
+
+```python
+xla_env = torch_xla2.default_env()
```

Comment:
QQ: Do we need to set env JAX_PLATFORM to specify the xla device?

Reply:
yes if you want to force CPU; otherwise it goes to the default.